CHANGELOG.md

```diff
@@ -6505,7 +6537,8 @@ Note: There was an issue publishing 1.12.0. Upgrade to 1.12.1.
 [v13.0.0]: https://github.com/nasa/cumulus/compare/v12.0.2...v13.0.0
 [v12.0.2]: https://github.com/nasa/cumulus/compare/v12.0.1...v12.0.2
 [v12.0.1]: https://github.com/nasa/cumulus/compare/v12.0.0...v12.0.1
-[v12.0.0]: https://github.com/nasa/cumulus/compare/v11.1.5...v12.0.0
+[v12.0.0]: https://github.com/nasa/cumulus/compare/v11.1.7...v12.0.0
+[v11.1.7]: https://github.com/nasa/cumulus/compare/v11.1.5...v11.1.7
```

Comment:
Do we need to include a note about why there's no 11.1.6? I'm not sure if that would cause confusion down the line or not.

Reply:
I think it's fine to skip versions.